pkgs/development/python-modules/aws-sam-translator/default.nix

```diff
@@ -48,6 +48,7 @@ buildPythonPackage rec {
   disabledTests = [
     # AssertionError: Expected 7 errors, found 9:
     "test_errors_13_error_definitionuri"
+    "test_region_configuration"
```

Comment:
Can you add comment with reason why this test is being disabled?

Reply:
Added a comment

Reply:
thanks. Please squash both commits into one and I think this can be merged.